spec/decorators/user_decorator_spec.rb

```diff
@@ -270,9 +270,9 @@
       create(:event, user: user, event_type: :email_changed, created_at: Time.zone.now - 30.days)
     end
 
-    it 'interleaves identities and events, decorates them, and sorts them in descending order' do
+    it 'interleaves identities and events, decorates the events, and sorts them in descending order' do
       expect(decorated_user.recent_events).
-        to eq [another_event.decorate, identity.decorate, event.decorate]
+        to eq [another_event.decorate, identity, event.decorate]
```

Comment:
This is a bummer IMO, seems weird that we mix decorated and not-decorated objects. Would you want to remove those other decorators as well, or just the identity one?

Reply:
I'd be fine removing the other ones too, but I'm less familiar with it. In this case pretty much every usage of `Identity` was decorated, and it even overloaded the `created_at` method.

This part was a bit odd to me at first, but it doesn't feel much weirder than putting two pretty distinct types of objects in an array. As long as they respond to the methods that they need to?